File: docs/components_page/components/card/header_footer.R
library(dashBootstrapComponents)

card <- dbcCard(
  list(
    dbcCardHeader("This is the header"),
    dbcCardBody(
      list(
        h4("Card title", className = "card-title"),
        p("This is some card text", className = "card-text")
      )
    ),
    dbcCardFooter("This is the footer")
  ),
  style = list(width = "18rem")
)

File: docs/components_page/components/card/header_footer.jl
using DashBootstrapComponents

card = dbc_card(
    [
        dbc_cardheader("This is the header"),
        dbc_cardbody([
            html_h4("Card title", className = "card-title"),
            html_p("This is some card text", className = "card-text"),
        ]),
        dbc_cardfooter("This is the footer"),
    ],
    style = Dict("width" => "18rem"),
);
